x-pack/platform/plugins/shared/cases/server/routes/api/cases/get_case_metadata.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import type { caseDomainV1 } from '../../../../common/types/domain';
+import { CASE_DETAILS_URL } from '../../../../common/constants';
+import { createCaseError } from '../../../common/error';
+import { createCasesRoute } from '../create_cases_route';
+import { DEFAULT_CASES_ROUTE_SECURITY } from '../constants';
+
+const params = {
+  params: schema.object({
+    case_id: schema.string(),
+  }),
+};
+
+export const getCaseMetadataRoute = () =>
+  createCasesRoute({
+    method: 'get',
+    path: `${CASE_DETAILS_URL}/metadata`,
+    security: DEFAULT_CASES_ROUTE_SECURITY,
+    params,
+    routerOptions: {
+      access: 'public',
```

Comment:
Is this being surfaced for users? If not, I would make this an internal api rather than public because I expect the schema will change as additional metadata may be added. Something akin to `{ alerts: AlertsMetadata, files: FileMetadata }` etc...

Reply:
Yes, I'll revert [this commit](https://github.com/elastic/kibana/pull/230192/commits/4c8a5ea48433847ae92a056c6906109eccaf8975) once the PR is ready to be merged.